Medium/sample.ipynb

```diff
@@ -0,0 +1,296 @@
+{
+ "cells": [
```

Comment:
@MSaiKiran9 The file name has to be renamed according to the naming convention.
Have a look [here](https://github.com/jupyter-naas/awesome-notebooks/blob/master/CONTRIBUTING.md).